radio/src/opentx.cpp

```diff
@@ -136,7 +136,7 @@ void per10ms()
   if (IS_ROTARY_ENCODER_NAVIGATION_ENABLE()) {
     static rotenc_t rePreviousValue;
     rotenc_t reNewValue = (ROTARY_ENCODER_NAVIGATION_VALUE / ROTARY_ENCODER_GRANULARITY);
-    int8_t scrollRE = reNewValue - rePreviousValue;
+    rotenc_t scrollRE = reNewValue - rePreviousValue;
```

Comment:
Please note that this one is probably not necessary (works either way), but I found it cleaner to use the same type here.

Reply:
TODO: check if we use this code on a 8bits platform (we are in an interrupt) and in this case what is rotenc_t type

Reply:
There is a define for that, it's basically 8bit an anything but ARM.